<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>my_Properties</class>
 <widget class="QDialog" name="my_Properties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(69, 105, 144);</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>531</width>
     <height>551</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>731</x>
     <y>130</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(119, 141, 169);
</string>
   </property>
   <property name="text">
    <string>buy house</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>731</x>
     <y>190</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(119, 141, 169);
</string>
   </property>
   <property name="text">
    <string>buy hotel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>731</x>
     <y>260</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(119, 141, 169);
</string>
   </property>
   <property name="text">
    <string>sell  property</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>731</x>
     <y>320</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(119, 141, 169);
</string>
   </property>
   <property name="text">
    <string>sell house</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>731</x>
     <y>380</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(119, 141, 169);
</string>
   </property>
   <property name="text">
    <string>sell hotel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>731</x>
     <y>70</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(119, 141, 169);
</string>
   </property>
   <property name="text">
    <string>show document</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>731</x>
     <y>440</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(119, 141, 169);
</string>
   </property>
   <property name="text">
    <string>mortgage</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
